frontend/simple/js/backend/hapi.js

```diff
@@ -8,8 +8,8 @@ import {sign} from '../../../../shared/functions'
 import * as Events from '../../../../shared/events'
 import {RESPONSE_TYPE} from '../../../../shared/constants'
 
-const {HashableEntry, HashableAction} = Events
-
+const {HashableEntry} = Events
+/* global fetch */
```

Comment:
Recommend doing either one of two things with this:

- Annotations for linters like this are canonically placed at the very top of the file, so either move it up there to the 1st line
- Or, place it in the standard config section in package.json, which has support for a special `globals` key (see docs)